app/common/configuration/config.go

```diff
@@ -0,0 +1,57 @@
+package configuration
+
+import (
+	"fmt"
+	"github.com/kelseyhightower/envconfig"
+)
+
+type Config struct {
+	API      APIConfig
+	Postgres PostgresConfig
```

Comment:
missing some configs that might be interesting like log level and the secret for token generation